components/index.html

```diff
@@ -0,0 +1,32 @@
+<!doctype html>
```

Comment:
I'd actually like to offload this to the server if we can.
I had tried this once before and opted to remove it because it was unwieldy managing the links.

Reply:
Resolved in 3a2d893